src/compat/posix/include/fcntl.h

```diff
@@ -49,6 +49,7 @@ extern int fcntl(int fd, int cmd, ...);
 #define F_DUPFD            12
 
 /* fcntl flags */
+#define O_ACCESS_MASK      0x3
```

Comment:
It's not a big problem, but I recommend to define it as (O_RDONLY | O_WRONLY | O_RDWR) instead.